scripts/gis-material-update/process/modules/tram_lines.py

```diff
@@ -10,8 +10,11 @@
 from modules.config import Config
 from modules.gis_processing import GisProcessor
 
-TRAM_ROUTE_TYPE = 0
+# Select only following route_type values:
+# 0 = Tram, Streetcar, Light rail. Any light rail or street level system within a metropolitan area
+# 900 = Tram Service
```

Comment:
It would be useful to include how HSL uses these codes. I.e. 0 = urban tram, 900 = light rail.

Reply:
Yes, I will change that. Now those code definition are directly from GTFS.